openapi/errors.go

```diff
@@ -75,4 +75,6 @@ var TransientErrorRegexes = []*regexp.Regexp{
 	regexp.MustCompile(`Unknown worker environment`),
 	regexp.MustCompile(`ClusterNotReadyException`),
 	regexp.MustCompile(`worker env .* not found`),
+	regexp.MustCompile(`Timed out after `),
+	regexp.MustCompile(`deadline exceeded`),
```

Comment:
Where does this one come from btw?

Reply:
First one - from permissions as I remember, 2nd one - from SCIM as I remember